plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -391,31 +401,53 @@ protected String createTableSql(RemoteTableName remoteTableName, List<String> co
     public Connection getConnection(ConnectorSession session, JdbcOutputTableHandle handle)
             throws SQLException
     {
-        return configureConnectionTransactionIsolation(super.getConnection(session, handle));
+        return configureSnapshotTransactionIsolation(super.getConnection(session, handle));
```

Comment:
I am fine with the new name, but if you keep the old one `configureConnectionTransactionIsolation`, you have a smaller diff. Up to you.